src/main.ts

```diff
@@ -336,4 +332,58 @@ program
     pluginUtil.init();
   });
 
+program
```

Comment:
please also add a `reset-config [name]` to reset all or one of the config fields